README.md

```diff
@@ -1,2 +1,55 @@
-# Ruby_Capstone
-Create a console app that will help you to keep a record of different types of things you won: books, music albums, movies, and games.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Catalog of my things
+> A ruby app that allows you to create Books, Music Albums, and Games objects and add a genre, author, and label to them.
+
+## Built With
+- Ruby
+- SQL
+
+## Video Presentation
+[Here](https://drive.google.com/file/d/1tYle1-7evN_f-9wVgAYk-voKYlEBRPCF/view?usp=sharing)
```

Comment:
- [ ] **Required:** The video link is not working. It shows the screenshot below.

![image](https://user-images.githubusercontent.com/27709832/183127663-688846c6-34b8-4e6e-a12a-389cf82c8857.png)